Compiler.cmake

```diff
@@ -11,6 +11,7 @@
 # Input Variables
 # * COMMON_MINIMUM_GCC_VERSION check for a minimum gcc version, default 4.4
 # * COMMON_USE_CXX03 When set, do not enable C++11 language features
+# * COMMON_DISABLE When set, do not set any flags
```

Comment:
There is an underlying bigger problem: In dependent projects, some flags have to be the same. This is the reason that for example COMMON_USE_CXX03 is a global option.

I would rather find a way to optionally disable 'safe' flags such as some warnings on a per-project level.

Can you give a list of flags which are problematic for you?